test/end2end.py

```diff
@@ -64,7 +64,9 @@ def setUp(self):
         # add /bin to $PATH, /lib to $PYTHONPATH
         self.topdir = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
         self.mympiscript = os.path.join(os.path.join(self.topdir, 'bin'), 'mympirun.py')
-        os.environ['PYTHONPATH'] = '%s:%s' % (os.path.join(self.topdir, 'lib'), os.getenv('PYTHONPATH', ''))
+        lib = os.path.join(self.topdir, 'lib')
+        eggs = os.path.join(self.topdir, '.eggs')
```

Comment:
maybe this should be the `lib/python<version>/site-packages` subdir?

to figure out the version, use `sys.version` (and drop the last digit, i.e. use `2.7`, not `2.7.10`)